packages/sensors_plus/tizen/src/sensors_plus_plugin.cc

```diff
@@ -67,40 +72,113 @@ class SensorsPlusPlugin : public flutter::Plugin {
   virtual ~SensorsPlusPlugin() {}
 
  private:
-  void SetupEventChannels(flutter::PluginRegistrar *registrar) {
-    std::unique_ptr<FlEventChannel> accelerometer_channel =
-        std::make_unique<FlEventChannel>(
-            registrar->messenger(),
-            "dev.fluttercommunity.plus/sensors/accelerometer",
+  void SetUpEventChannels(flutter::PluginRegistrar *registrar,
+                          SensorsPlusPlugin *plugin) {
+    std::unique_ptr<FlMethodChannel> method_channel =
+        std::make_unique<FlMethodChannel>(
+            registrar->messenger(), "dev.fluttercommunity.plus/sensors/method",
```

Comment:
nit: This method (`SetUpEventChannels`) is a place for initializing event channels. Initializing the method channel could be done outside (in `RegisterWithRegistrar`).

Reply:
Thanks !